SABnzbd.py

```diff
@@ -431,10 +431,7 @@ def print_modules():
     if sabnzbd.decoder.HAVE_YENC:
         logging.info("_yenc module... found!")
     else:
-        if hasattr(sys, "frozen"):
-            logging.error(T('_yenc module... NOT found!'))
-        else:
-            logging.info("_yenc module... NOT found!")
+        logging.warning(T('_yenc module... NOT found!'))
```

Comment:
It's still an error to miss _yenc in a "frozen" app.
Are you sure about the warning? First time in 10 years.
If you have no choice, it's no fun to get a warning every time.
But then again, it is way better to have it.